cli/tests/testdata/test/pass.ts

```diff
@@ -35,3 +35,11 @@ Deno.test("test\r", () => {
 Deno.test("test\v", () => {
   console.error("console.error");
 });
+
+const testFn: () => never = () => {
+  throw new Error("boom!");
+};
+
+(testFn as any)[Symbol.for("Deno.test.location")] = "foo/bar/fizz.ts:100:200";
```

Comment:
Ok awesome! I was a little worried about using `any` here, but that context makes a lot of sense

Reply:
I was wrong here - the value here should be:
```
{
  fileName: "file://foo/bar/fizz.ts",
  lineNumber: 100,
  columnNumber: 200
}
```

Reply:
Also, we should move this test to `cli/tests/testdata/test/fail.ts` since this file is expected to pass fine :) please add a comment that we're testing overriding test location here.